src/components/ChoiceGroup/ChoiceGroup.tsx

```diff
@@ -0,0 +1,99 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import '../../yamui';
+import * as React from 'react';
+import { BaseComponentProps } from '../../util/BaseComponent/props';
+import { ChoiceGroup as FabricChoiceGroup, IChoiceGroupOption } from 'office-ui-fabric-react/lib/ChoiceGroup';
+// prettier-ignore
+import {
+  IChoiceGroupOptionProps,
+} from 'office-ui-fabric-react/lib/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.types';
+import { getStyles, getOptionStyles } from './ChoiceGroup.styles';
+import { join } from '../../util/classNames';
+import Text from '../Text';
+
+export interface ChoiceGroupOption {
+  /**
+   * A required key to uniquely identify the option.
+   */
+  key: string;
+
+  /**
+   * Text for the option.
+   */
+  text: string;
+
+  /**
+   * Label text for the option.
+   */
+  label?: string;
+}
+
+export interface ChoiceGroupProps extends BaseComponentProps {
+  /**
+   * Items to be presented to the user.
+   */
+  options: ChoiceGroupOption[];
+
+  /**
+   * Label text that will be rendered above the choices.
+   */
+  label?: string;
+
+  /**
+   * The selected option.
+   */
+  selectedKey?: ChoiceGroupOption['key'];
+
+  /**
+   * Function to call when user changes the selected item.
+   */
+  onChange(key: ChoiceGroupOption['key']): void;
+}
+
+const getTextWithLabel = (label: string, sublabel: string) => {
+  return () => (
+    <span className="ms-Label">
+      <Text bold={true}>{`${label}:`}</Text>
+      <Text>&nbsp;{sublabel}</Text>
+    </span>
+  );
+};
+
+const mapOption = (option: ChoiceGroupOption): IChoiceGroupOptionProps => {
+  const { text, label } = option;
+  const onRenderLabel = label ? getTextWithLabel(text, label) : undefined;
```

Comment:
i wouldn't make this conditional, i'd always use our onRenderLabel method and handle the bold label conditionally in it. then the markup/styling will be consistent in both cases, less chance for Fabric to break our visual diffs in the future, slightly simpler debugging